src/styles/main.scss

```diff
@@ -1,3 +1,81 @@
+$columns: 7;
+$gap: 1px;
+$square-size: 100px;
+$calendar-padding: 5px;
+// '--month-length_31' 31,
+// '--month-length_30' 30,
+// '--month-length_29' 29,
+// '--month-length_28' 28;
+$month-length: 29;
+// '--start-day_mon' 1,
+// '--start-day_tue' 2,
+// '--start-day_wed' 3,
+// '--start-day_thu' 4,
+// '--start-day_fri' 5,
+// '--start-day_sat' 6,
+// '--start-day_sun' 7;
```

Comment:
```suggestion
$month-length: 31;
```
remove commented code, month length set 31 (max possible month length)

Reply:
Thank you

I found the solution but I am getting this error:

![image](https://user-images.githubusercontent.com/126076069/232480807-f5c606ca-5a70-4105-9237-5dd9ca3936b5.png)


I have on block calendar with 2 modifiers,
and the rest have an element and a modifier.

could you please help me?

Reply:
as you can see the elements are inside the block